EIPS/eip-721.md

```diff
@@ -440,7 +440,7 @@ XXXXERC721, by William Entriken -- a scalable example implementation
 1. Curio Cards. https://mycuriocards.com
 1. Rare Pepe. https://rarepepewallet.com
 1. Auctionhouse Asset Interface. https://github.com/dob/auctionhouse/blob/master/contracts/Asset.sol
-1. OpenZeppelin SafeERC20.sol Implementation. https://github.com/OpenZeppelin/zeppelin-solidity/blob/master/contracts/token/ERC20/SafeERC20.sol
```

Comment:
I wonder if we could, instead of link to new link, consider find the permlink to old file and link to that permlink?

@SamWilsn @Pandapip1 for thoughts

Reply:
That works for me, have a specific version to propose?

Reply:
> That works for me, have a specific version to propose?

You are the author, so I'd probably the best version would be the intended version you linkes to when you were authoring this EIP.

Reply:
How about we just delete the link altogether? It doesn't add much value.

Reply:
As a peer author reviewing this EIP, I think such links serve a lot of value collecting landscape of adoptions and make techinical arguments more convincing.